src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import './Calculator.css';
+import CalcButton from './CalcButton';
+
+function Calculator() {
+  return (
+    <div className="Calculator">
+      <div className="Calculator-result">0</div>
+
+      <CalcButton character="AC" />
+      <CalcButton character="+/-" />
+      <CalcButton character="%" />
+      <CalcButton character="+" classButton="orangeButton" />
+
+      <CalcButton character="7" />
+      <CalcButton character="8" />
+      <CalcButton character="9" />
+      <CalcButton character="x" classButton="orangeButton" />
+
+      <CalcButton character="4" />
+      <CalcButton character="5" />
+      <CalcButton character="6" />
+      <CalcButton character="-" classButton="orangeButton" />
+
+      <CalcButton character="1" />
+      <CalcButton character="2" />
+      <CalcButton character="3" />
+      <CalcButton character="/" classButton="orangeButton" />
```

Comment:
- [ ] I think you have interchanged the position of `+` and `÷` text. Also, you can use the `÷` symbol for division, I know it's not on the keyboard but you can search for `division text` on google or you can use `HTML character`